FourSquare/Controllers/Background/BackgroundViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  BackgroundViewController.swift
+//  FourSquare
+//
+//  Created by Mylo Ho on 8/1/16.
+//  Copyright © 2016 Le Van Binh. All rights reserved.
+//
+
+import UIKit
+import LGSideMenuController
+import SwiftUtils
+
+class BackgroundViewController: LGSideMenuController {
+
+    var leftViewController = LeftSideMenuViewController()
+
+    var allMenuItems: [ItemMenu] = []
+
+    var activeMenuItems: [ItemMenu] = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.leftViewSwipeGestureEnabled = false
+        for i in 0..<6 {
```

Comment:
make a function for initializing menu items.
remember to break down code into functions